src/vs/workbench/contrib/notebook/browser/view/notebookCellListView.ts

```diff
@@ -276,9 +276,14 @@ export class NotebookCellListView<T> extends ListView<T> {
 	insertWhitespace(afterPosition: number, size: number): string {
 		const scrollTop = this.scrollTop;
 		const id = `${++this._lastWhitespaceId}`;
+		const previousRenderRange = this.getRenderRange(this.lastRenderTop, this.lastRenderHeight);
```

Comment:
a new `getCurrentRenderRange` could also be introduced since these parameters belong to the listview anyway.